src/app/components/navbar/components/message/message.component.ts

```diff
@@ -14,30 +15,16 @@ export class MessageComponent implements OnInit {
   constructor(private messages: MessagesService) { }
 
   ngOnInit() {
-    this.messages.getStatus()
-      .then(message => {
-        this.message = message;
-        const lastMessage = localStorage.getItem('maintenance');
-
-        if (!lastMessage || this.message.id !== lastMessage) {
-          this.open();
-        }
-      })
-      .catch(e => {
-        // FIXME: Suppress the error until we design a better backend solution
-      });
-  }
-
-  open() {
-    this.showing = true;
-  }
-
-  close() {
-    this.showing = false;
-    localStorage.setItem('maintenance', this.message.id);
-  }
-
-  toggle() {
-    this.showing = !this.showing;
+    if (environment.production) {
+      setInterval(async () => {
+        this.messages.getStatus().then(message => {
+          this.message = message;
+          this.showing = this.message.isUnderMaintenance;
+        })
+        .catch (e => {
```

Comment:
Since `e` is a throw-away property here, can we rename it to `_`?
```suggestion
        .catch (_ => {
```